ci-operator/config/openshift/release/openshift-release-master__nightly-4.13.yaml

```diff
@@ -638,6 +638,12 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-gcp-realtime
+- as: e2e-gcp-ovn-xpn
+  interval: 168h
```

Comment:
Ideally I would like if we could bump this interval to ideally `8h`. I see that all the other intervals are running much less frequently. We would like to start getting regular periodics for this feature which we are trying to release in 4.13, but we could dial it back once we get a better signal or release.